### PR TITLE
[Leap] Make message to ignore built-in library method C# specific

### LINKED ISSUE #610: Consider calling out DateTime.IsLeapYear() for Leap exercise
Was doing some mentoring and noticed our Leap instructions just say 

>If your language provides a method in the standard library that does this look-up, pretend it doesn't exist and implement it yourself.

I think since we have the ability to be specific, since the README exists within the C# repo, we can explicitly call out DateTime.IsLeapYear().  I think it'd be a good learning opportunity, highlighting this function exists in the first place. As well as makes it a bit more personable to the C# track.

### PR DESCRIPTION
**Pull request by robkeim**
Closes #610

@jpreese FYI

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Just waiting for the CI to become green!

**Comment by ErikSchierboom** [comment]
I don't know if there are other ways to add to the README. Maybe @kytrinyx knows?

And so far, we haven't yet been regenerating the README's, but we should be able to do that using the `configlet` command.

**Comment by ErikSchierboom** [comment]
@robkeim According to [the source](https://github.com/exercism/configlet/blob/master/track/exercise_readme.go), it looks like we should still be using the `hints.md` file (which name is a bit unfortunate in this case). Could you add the new text to the `hints.md` file, and then regenerate the README's using configlet?

**Comment by robkeim** [comment]
@ErikSchierboom yeah I'll go ahead and make that change

File: exercises/leap/.meta/hints.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Hints
```
Comment (ErikSchierboom) [inline exercises/leap/.meta/hints.md]: I was thinking that perhaps this should be something else in this case, maybe "Notes"  or something like that?

Comment (robkeim) [reply exercises/leap/.meta/hints.md]: Makes sense to me... fixed!

**Comment by ErikSchierboom** [comment]
Waiting for the CI.

**Comment by ErikSchierboom** [comment]
Thanks!

**Pull request merged**